games-mud/tkmoo/tkmoo-0.3.32-r1.ebuild

```diff
@@ -1,44 +1,45 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
-inherit eutils
+EAPI=8
 
-MY_PN=${PN/moo/MOO-light}
-MY_P=${P/moo/MOO-light}
+inherit desktop
+
+MY_PN="${PN/moo/MOO-light}"
+MY_P="${P/moo/MOO-light}"
 DESCRIPTION="MOO Client written in Tcl/Tk"
 HOMEPAGE="http://www.awns.com/tkMOO-light/"
 SRC_URI="http://www.awns.com/tkMOO-light/Source/${MY_P}.tar.gz"
 
 LICENSE="tkMOO"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
 
 DEPEND="
-	>=dev-lang/tcl-8.3.3:0=
-	>=dev-lang/tk-8.3.3:0="
-RDEPEND=${DEPEND}
+	dev-lang/tcl:0=
+	dev-lang/tk:0="
```

Comment:
The 0 slot isn't needed anymore either.

Reply:
On a side-note, I meant to say only the 0 while keeping :=, however upon closer look this doesn't link with the libraries so the := really isn't needed and this is fine :)

Reply:
Oh i see, good to know.